src/main/java/org/skriptlang/skriptworldguard/worldguard/RegionUtils.java

```diff
@@ -136,9 +137,39 @@ public static RegionContainer getRegionContainer() {
 	 * - region data for the given world has failed to load
 	 * - support for regions has been disabled
 	 */
-	@Nullable
-	public static RegionManager getRegionManager(World world) {
+	public static @Nullable RegionManager getRegionManager(World world) {
 		return WorldGuard.getInstance().getPlatform().getRegionContainer().get(BukkitAdapter.adapt(world));
 	}
 
+	/**
+	 * Get aall regions of all worlds
+	 * @return Array of {@link WorldGuardRegion}
+	 */
+	public static WorldGuardRegion @Nullable [] getRegions() {
+		return getRegions(null);
+	}
+
+	/**
+	 * Get all regions of all worlds or provide {@code world} to get all regions of that {@link World}
+	 * @param world The {@link World} to get regions from, or {@code null} to get regions from all worlds
+	 * @return Array of {@link WorldGuardRegion}

```

Comment:
```suggestion
	 * @return All regions from the world, or all worlds if {@code world} is null.
```
the return type is already specified on the line below, `@return` is usually just snappy description of what you return exactly